scss/_patterns_rule.scss

```diff
@@ -16,14 +16,6 @@
   }
 
   .p-rule--highlight {
-    @include vf-highlight-bar($colors--light-theme--text-default);
-
-    &.is-dark {
-      @include vf-highlight-bar($colors--dark-theme--text-default);
-    }
-
-    &.is-accent {
-      @include vf-highlight-bar($color-accent);
-    }
+    @include vf-highlight-bar($colors--theme--text-default);
```

Comment:
You didn't have to do this, thanks :)

I think by mistake you removed the `.is-accent` version. I don't think we have separate themed accent colour, so for now just keep it as is, regardless of the theme.